app/models/card.rb

```diff
@@ -1,5 +1,7 @@
 class Card < ActiveRecord::Base
+  has_attached_file :image, :styles => { medium: "300x300>"}, default_url: "/images/missing.png"
```

Comment:
Line is too long. [96/80]<br>Use the new Ruby 1.9 hash syntax.<br>Space inside } missing.